README.md

```diff
@@ -12,25 +12,29 @@ As mentioned in the [FAQ](https://www.infracost.io/docs/faq), you can run Infrac
 
 Infracost should be run using the [Terraform directory method](https://www.infracost.io/docs/#1-terraform-directory) with this GitHub Action. Once [this issue](https://github.com/infracost/infracost/issues/394) is released, we'll be able to support other methods.
```

Comment:
We should remove this, not sure if we should add a sentence here to say either path or config_file is required like we do below.